Convert lastModified() calls to the more precise getModifiedTime()

cunei opened the pull request:
Use the new native file modification timestamps, which return times with full millisecond precisions whenever possible. Depends on https://github.com/sbt/io/pull/92

Commit:
Convert lastModified() calls to sbt.io.Milli.getModifiedTime()

typesafe-tools commented:
To validate this pull request against other sbt modules, please visit <A HREF="https://jenkins.scala-sbt.org:8592/job/sbt-validator/parambuild/?util=pull/134/head&deploy_to_bintray=false&run_sbt_community=false">this link.</A>

typesafe-tools commented:
A validation involving this pull request <a href="https://jenkins.scala-sbt.org:8592/job/sbt-validator/294/console">is in progress...</a>

typesafe-tools commented:
[//]: # (Refs: pull/3789/head-pull/463/head-pull/92/head-pull/189/head-pull/134/head-1.x)
The validator <A HREF="https://jenkins.scala-sbt.org:8592/job/sbt-validator/294/">has checked</A> the following projects against Scala 2.12,
tested using dbuild, projects built on top of each other.

Project             | Reference            | Commit
:------------------ | :------------------- | :----------------
  sbt               | <A HREF="https://github.com/sbt/sbt/pull/3789">pull/3789/head      </A> | sbt/sbt@66763f9a542b6fa28938e38dd480261cd0e1457b
  zinc              | <A HREF="https://github.com/sbt/zinc/pull/463">pull/463/head       </A> | sbt/zinc@e809ae51b11e1af3df66e83a16f5c20012203b70
  io                | <A HREF="https://github.com/sbt/io/pull/92">pull/92/head        </A> | sbt/io@04a902a982fdd702e8104d99c71050cb32e20c8a
  librarymanagement | <A HREF="https://github.com/sbt/librarymanagement/pull/189">pull/189/head       </A> | sbt/librarymanagement@bc255105b127cb8857a21528535557565cd96701
  util              | <A HREF="https://github.com/sbt/util/pull/134">pull/134/head       </A> | sbt/util@3b404aecaa8e515a7c8f102241d4ab159c171e91
  website           | <A HREF="https://github.com/sbt/website/tree/1.x">1.x                 </A> | 

:x: The result is: ***FAILED***
<A HREF="https://jenkins.scala-sbt.org:8592/job/sbt-validator/parambuild?sbt=pull/3789/head&zinc=pull/463/head&io=pull/92/head&librarymanagement=pull/189/head&util=pull/134/head&website=1.x&deploy_to_bintray=false&run_sbt_community=false">(restart)</A>

Commit:
Moved Milli._ to IO.

typesafe-tools commented:
A validation involving this pull request <a href="https://jenkins.scala-sbt.org:8592/job/sbt-validator/299/console">is in progress...</a>

typesafe-tools commented:
[//]: # (Refs: pull/3789/head-pull/463/head-pull/92/head-pull/189/head-pull/134/head-1.x)
The validator <A HREF="https://jenkins.scala-sbt.org:8592/job/sbt-validator/299/">has checked</A> the following projects against Scala 2.12,
tested using dbuild, projects built on top of each other.

Project             | Reference            | Commit
:------------------ | :------------------- | :----------------
  sbt               | <A HREF="https://github.com/sbt/sbt/pull/3789">pull/3789/head      </A> | sbt/sbt@32846825920e5a79a8a209d784d10711b0d2b1fc
  zinc              | <A HREF="https://github.com/sbt/zinc/pull/463">pull/463/head       </A> | sbt/zinc@5d07d2994db55efa325e12bd90556cea86a31804
  io                | <A HREF="https://github.com/sbt/io/pull/92">pull/92/head        </A> | sbt/io@e9400ca4cf2ff9f748f8ce60415270302ff88b2e
  librarymanagement | <A HREF="https://github.com/sbt/librarymanagement/pull/189">pull/189/head       </A> | sbt/librarymanagement@bc255105b127cb8857a21528535557565cd96701
  util              | <A HREF="https://github.com/sbt/util/pull/134">pull/134/head       </A> | sbt/util@35cd61c562915de7422d1510273c7b537e42ef27
  website           | <A HREF="https://github.com/sbt/website/tree/1.x">1.x                 </A> | 

:x: The result is: ***FAILED***
<A HREF="https://jenkins.scala-sbt.org:8592/job/sbt-validator/parambuild?sbt=pull/3789/head&zinc=pull/463/head&io=pull/92/head&librarymanagement=pull/189/head&util=pull/134/head&website=1.x&deploy_to_bintray=false&run_sbt_community=false">(restart)</A>

typesafe-tools commented:
A validation involving this pull request <a href="https://jenkins.scala-sbt.org:8592/job/sbt-validator/301/console">is in progress...</a>

typesafe-tools commented:
[//]: # (Refs: pull/3789/head-pull/463/head-pull/92/head-pull/189/head-pull/134/head-1.x)
The validator <A HREF="https://jenkins.scala-sbt.org:8592/job/sbt-validator/301/">has checked</A> the following projects against Scala 2.12,
tested using dbuild, projects built on top of each other.

Project             | Reference            | Commit
:------------------ | :------------------- | :----------------
  sbt               | <A HREF="https://github.com/sbt/sbt/pull/3789">pull/3789/head      </A> | sbt/sbt@32846825920e5a79a8a209d784d10711b0d2b1fc
  zinc              | <A HREF="https://github.com/sbt/zinc/pull/463">pull/463/head       </A> | sbt/zinc@5d07d2994db55efa325e12bd90556cea86a31804
  io                | <A HREF="https://github.com/sbt/io/pull/92">pull/92/head        </A> | sbt/io@07ef05614c769f9ddea6ad8eb783eb59a416b79a
  librarymanagement | <A HREF="https://github.com/sbt/librarymanagement/pull/189">pull/189/head       </A> | sbt/librarymanagement@74380f09b655c2653fbd7888600c6585d94e8959
  util              | <A HREF="https://github.com/sbt/util/pull/134">pull/134/head       </A> | sbt/util@35cd61c562915de7422d1510273c7b537e42ef27
  website           | <A HREF="https://github.com/sbt/website/tree/1.x">1.x                 </A> | 

:x: The result is: ***FAILED***
<A HREF="https://jenkins.scala-sbt.org:8592/job/sbt-validator/parambuild?sbt=pull/3789/head&zinc=pull/463/head&io=pull/92/head&librarymanagement=pull/189/head&util=pull/134/head&website=1.x&deploy_to_bintray=false&run_sbt_community=false">(restart)</A>

typesafe-tools commented:
A validation involving this pull request <a href="https://jenkins.scala-sbt.org:8592/job/sbt-validator/302/console">is in progress...</a>

typesafe-tools commented:
[//]: # (Refs: pull/3789/head-pull/463/head-pull/92/head-pull/189/head-pull/134/head-1.x)
The validator <A HREF="https://jenkins.scala-sbt.org:8592/job/sbt-validator/302/">has checked</A> the following projects against Scala 2.12,
tested using dbuild, projects built on top of each other.

Project             | Reference            | Commit
:------------------ | :------------------- | :----------------
  sbt               | <A HREF="https://github.com/sbt/sbt/pull/3789">pull/3789/head      </A> | sbt/sbt@a0c884deda2d9330fd549d421438210b5a473833
  zinc              | <A HREF="https://github.com/sbt/zinc/pull/463">pull/463/head       </A> | sbt/zinc@422340c8e0abb31a0cc42c9e3ff3c726498c8028
  io                | <A HREF="https://github.com/sbt/io/pull/92">pull/92/head        </A> | sbt/io@e433a5c62aa94dea4a1203c2c29af97e0d396afc
  librarymanagement | <A HREF="https://github.com/sbt/librarymanagement/pull/189">pull/189/head       </A> | sbt/librarymanagement@aacab10c79abbbe206ab7b6e251a6ab8e6313fdd
  util              | <A HREF="https://github.com/sbt/util/pull/134">pull/134/head       </A> | sbt/util@d03dfb39817be24a84556cabfc4affc4ac257bb2
  website           | <A HREF="https://github.com/sbt/website/tree/1.x">1.x                 </A> | 

:white_check_mark: The result is: ***SUCCESS***
<A HREF="https://jenkins.scala-sbt.org:8592/job/sbt-validator/parambuild?sbt=pull/3789/head&zinc=pull/463/head&io=pull/92/head&librarymanagement=pull/189/head&util=pull/134/head&website=1.x&deploy_to_bintray=false&run_sbt_community=false">(restart)</A>

typesafe-tools commented:
A validation involving this pull request <a href="https://jenkins.scala-sbt.org:8592/job/sbt-validator/303/console">is in progress...</a>

typesafe-tools commented:
[//]: # (Refs: pull/3789/head-pull/463/head-pull/92/head-pull/189/head-pull/134/head-1.x)
The validator <A HREF="https://jenkins.scala-sbt.org:8592/job/sbt-validator/303/">has checked</A> the following projects against Scala 2.12,
tested using dbuild, projects built on top of each other.

Project             | Reference            | Commit
:------------------ | :------------------- | :----------------
  sbt               | <A HREF="https://github.com/sbt/sbt/pull/3789">pull/3789/head      </A> | sbt/sbt@a0c884deda2d9330fd549d421438210b5a473833
  zinc              | <A HREF="https://github.com/sbt/zinc/pull/463">pull/463/head       </A> | sbt/zinc@422340c8e0abb31a0cc42c9e3ff3c726498c8028
  io                | <A HREF="https://github.com/sbt/io/pull/92">pull/92/head        </A> | sbt/io@e433a5c62aa94dea4a1203c2c29af97e0d396afc
  librarymanagement | <A HREF="https://github.com/sbt/librarymanagement/pull/189">pull/189/head       </A> | sbt/librarymanagement@aacab10c79abbbe206ab7b6e251a6ab8e6313fdd
  util              | <A HREF="https://github.com/sbt/util/pull/134">pull/134/head       </A> | sbt/util@d03dfb39817be24a84556cabfc4affc4ac257bb2
  website           | <A HREF="https://github.com/sbt/website/tree/1.x">1.x                 </A> | 

:x: The result is: ***FAILED***
<A HREF="https://jenkins.scala-sbt.org:8592/job/sbt-validator/parambuild?sbt=pull/3789/head&zinc=pull/463/head&io=pull/92/head&librarymanagement=pull/189/head&util=pull/134/head&website=1.x&deploy_to_bintray=false&run_sbt_community=false">(restart)</A>

typesafe-tools commented:
A validation involving this pull request <a href="https://jenkins.scala-sbt.org:8592/job/sbt-validator/305/console">is in progress...</a>

typesafe-tools commented:
[//]: # (Refs: pull/3789/head-pull/463/head-1.1.x-pull/189/head-pull/134/head-1.x)
The validator <A HREF="https://jenkins.scala-sbt.org:8592/job/sbt-validator/305/">has checked</A> the following projects against Scala 2.12,
tested using dbuild, projects built on top of each other.

Project             | Reference            | Commit
:------------------ | :------------------- | :----------------
  sbt               | <A HREF="https://github.com/sbt/sbt/pull/3789">pull/3789/head      </A> | sbt/sbt@0f151e9f2c712a1af98a95596e51ff6ef6eeaf35
  zinc              | <A HREF="https://github.com/sbt/zinc/pull/463">pull/463/head       </A> | sbt/zinc@422340c8e0abb31a0cc42c9e3ff3c726498c8028
  io                | <A HREF="https://github.com/sbt/io/tree/1.1.x">1.1.x               </A> | sbt/io@0a8d26121d2ef03f9bc03d710baa6957e5ec6f03
  librarymanagement | <A HREF="https://github.com/sbt/librarymanagement/pull/189">pull/189/head       </A> | sbt/librarymanagement@aacab10c79abbbe206ab7b6e251a6ab8e6313fdd
  util              | <A HREF="https://github.com/sbt/util/pull/134">pull/134/head       </A> | sbt/util@d03dfb39817be24a84556cabfc4affc4ac257bb2
  website           | <A HREF="https://github.com/sbt/website/tree/1.x">1.x                 </A> | 

:white_check_mark: The result is: ***SUCCESS***

The binaries are at https://dl.bintray.com/sbt/maven-snapshots
The new sbt version is: 1.0.3-bin-20171207T013530
<A HREF="https://jenkins.scala-sbt.org:8592/job/sbt-validator/parambuild?sbt=pull/3789/head&zinc=pull/463/head&io=1.1.x&librarymanagement=pull/189/head&util=pull/134/head&website=1.x&deploy_to_bintray=false&run_sbt_community=false">(restart)</A>

Commit:
Allow FileInfo for non-existent files with the new timestamps

FileInfo is used to wrap information like last modified time on
files that may or may not exist. Arguably, that does not make
much sense: the non-existent files should not lead to modification
file information, hashes, and a persistent serialized version of
the resulting meaningless information. However, considering that
the FileInfo information is serialized and saved, it is necessary
to preserve compatibility at this stage. Therefore the modification
time is explicitly set to zero for those files that do not exist
when each FileInfo is built.

typesafe-tools commented:
A validation involving this pull request <a href="https://jenkins.scala-sbt.org:8592/job/sbt-validator/307/console">is in progress...</a>

typesafe-tools commented:
[//]: # (Refs: pull/3789/head-pull/463/head-1.x-pull/189/head-pull/134/head-1.x)
The validator <A HREF="https://jenkins.scala-sbt.org:8592/job/sbt-validator/307/">has checked</A> the following projects against Scala 2.12,
tested using dbuild, projects built on top of each other.

Project             | Reference            | Commit
:------------------ | :------------------- | :----------------
  sbt               | <A HREF="https://github.com/sbt/sbt/pull/3789">pull/3789/head      </A> | sbt/sbt@0f151e9f2c712a1af98a95596e51ff6ef6eeaf35
  zinc              | <A HREF="https://github.com/sbt/zinc/pull/463">pull/463/head       </A> | sbt/zinc@422340c8e0abb31a0cc42c9e3ff3c726498c8028
  io                | <A HREF="https://github.com/sbt/io/tree/1.x">1.x                 </A> | sbt/io@98cc14a6f7191ad30a0fefa60be43ae0eb0e35af
  librarymanagement | <A HREF="https://github.com/sbt/librarymanagement/pull/189">pull/189/head       </A> | sbt/librarymanagement@aacab10c79abbbe206ab7b6e251a6ab8e6313fdd
  util              | <A HREF="https://github.com/sbt/util/pull/134">pull/134/head       </A> | sbt/util@cd4346c5d7aa0c04893bc7ae85c240e1a6926c62
  website           | <A HREF="https://github.com/sbt/website/tree/1.x">1.x                 </A> | 

:x: The result is: ***FAILED***
<A HREF="https://jenkins.scala-sbt.org:8592/job/sbt-validator/parambuild?sbt=pull/3789/head&zinc=pull/463/head&io=1.x&librarymanagement=pull/189/head&util=pull/134/head&website=1.x&deploy_to_bintray=false&run_sbt_community=false">(restart)</A>

typesafe-tools commented:
A validation involving this pull request <a href="https://jenkins.scala-sbt.org:8592/job/sbt-validator/309/console">is in progress...</a>

typesafe-tools commented:
[//]: # (Refs: pull/3789/head-pull/463/head-1.1.x-pull/189/head-pull/134/head-1.x)
The validator <A HREF="https://jenkins.scala-sbt.org:8592/job/sbt-validator/309/">has checked</A> the following projects against Scala 2.12,
tested using dbuild, projects built on top of each other.

Project             | Reference            | Commit
:------------------ | :------------------- | :----------------
  sbt               | <A HREF="https://github.com/sbt/sbt/pull/3789">pull/3789/head      </A> | sbt/sbt@0f151e9f2c712a1af98a95596e51ff6ef6eeaf35
  zinc              | <A HREF="https://github.com/sbt/zinc/pull/463">pull/463/head       </A> | sbt/zinc@422340c8e0abb31a0cc42c9e3ff3c726498c8028
  io                | <A HREF="https://github.com/sbt/io/tree/1.1.x">1.1.x               </A> | sbt/io@0a8d26121d2ef03f9bc03d710baa6957e5ec6f03
  librarymanagement | <A HREF="https://github.com/sbt/librarymanagement/pull/189">pull/189/head       </A> | sbt/librarymanagement@aacab10c79abbbe206ab7b6e251a6ab8e6313fdd
  util              | <A HREF="https://github.com/sbt/util/pull/134">pull/134/head       </A> | sbt/util@cd4346c5d7aa0c04893bc7ae85c240e1a6926c62
  website           | <A HREF="https://github.com/sbt/website/tree/1.x">1.x                 </A> | 

:white_check_mark: The result is: ***SUCCESS***

The binaries are at https://dl.bintray.com/sbt/maven-snapshots
The new sbt version is: 1.0.3-bin-20171209T002807
<A HREF="https://jenkins.scala-sbt.org:8592/job/sbt-validator/parambuild?sbt=pull/3789/head&zinc=pull/463/head&io=1.1.x&librarymanagement=pull/189/head&util=pull/134/head&website=1.x&deploy_to_bintray=false&run_sbt_community=false">(restart)</A>

typesafe-tools commented:
A validation involving this pull request <a href="https://jenkins.scala-sbt.org:8592/job/sbt-validator/315/console">is in progress...</a>

typesafe-tools commented:
A validation involving this pull request <a href="https://jenkins.scala-sbt.org:8592/job/sbt-validator/316/console">is in progress...</a>

dwijnand committed:
Use IO.getModified over importing the method

.. and make getModifiedTimeOrZero private.

dwijnand reviewed (commented):
Just a couple of tweaks: https://github.com/cunei/util/pull/1

typesafe-tools commented:
[//]: # (Refs: pull/3789/head-pull/463/head-pull/100/head-pull/189/head-pull/134/head-1.x)
The validator <A HREF="https://jenkins.scala-sbt.org:8592/job/sbt-validator/316/">has checked</A> the following projects against Scala 2.12,
tested using dbuild, projects built on top of each other.

Project             | Reference            | Commit
:------------------ | :------------------- | :----------------
  sbt               | <A HREF="https://github.com/sbt/sbt/pull/3789">pull/3789/head      </A> | sbt/sbt@30b53618c99d80dae9886651436df233d8b4cb2f
  zinc              | <A HREF="https://github.com/sbt/zinc/pull/463">pull/463/head       </A> | sbt/zinc@422340c8e0abb31a0cc42c9e3ff3c726498c8028
  io                | <A HREF="https://github.com/sbt/io/pull/100">pull/100/head       </A> | sbt/io@8c5947be6c04c6a0b5d6543ec181f10f4b5d6cc7
  librarymanagement | <A HREF="https://github.com/sbt/librarymanagement/pull/189">pull/189/head       </A> | sbt/librarymanagement@aacab10c79abbbe206ab7b6e251a6ab8e6313fdd
  util              | <A HREF="https://github.com/sbt/util/pull/134">pull/134/head       </A> | sbt/util@cd4346c5d7aa0c04893bc7ae85c240e1a6926c62
  website           | <A HREF="https://github.com/sbt/website/tree/1.x">1.x                 </A> | 

:white_check_mark: The result is: ***SUCCESS***

The binaries are at https://dl.bintray.com/sbt/maven-snapshots
The new sbt version is: 1.0.3-bin-20171213T145141
<A HREF="https://jenkins.scala-sbt.org:8592/job/sbt-validator/parambuild?sbt=pull/3789/head&zinc=pull/463/head&io=pull/100/head&librarymanagement=pull/189/head&util=pull/134/head&website=1.x&deploy_to_bintray=false&run_sbt_community=false">(restart)</A>

typesafe-tools commented:
A validation involving this pull request <a href="https://jenkins.scala-sbt.org:8592/job/sbt-validator/320/console">is in progress...</a>

typesafe-tools commented:
[//]: # (Refs: pull/3789/head-pull/463/head-pull/100/head-pull/189/head-pull/134/head-1.x)
The validator <A HREF="https://jenkins.scala-sbt.org:8592/job/sbt-validator/320/">has checked</A> the following projects against Scala 2.12,
tested using dbuild, projects built on top of each other.

Project             | Reference            | Commit
:------------------ | :------------------- | :----------------
  sbt               | <A HREF="https://github.com/sbt/sbt/pull/3789">pull/3789/head      </A> | sbt/sbt@30b53618c99d80dae9886651436df233d8b4cb2f
  zinc              | <A HREF="https://github.com/sbt/zinc/pull/463">pull/463/head       </A> | sbt/zinc@422340c8e0abb31a0cc42c9e3ff3c726498c8028
  io                | <A HREF="https://github.com/sbt/io/pull/100">pull/100/head       </A> | sbt/io@8c5947be6c04c6a0b5d6543ec181f10f4b5d6cc7
  librarymanagement | <A HREF="https://github.com/sbt/librarymanagement/pull/189">pull/189/head       </A> | sbt/librarymanagement@aacab10c79abbbe206ab7b6e251a6ab8e6313fdd
  util              | <A HREF="https://github.com/sbt/util/pull/134">pull/134/head       </A> | sbt/util@cd4346c5d7aa0c04893bc7ae85c240e1a6926c62
  website           | <A HREF="https://github.com/sbt/website/tree/1.x">1.x                 </A> | 

:x: The result is: ***FAILED***
<A HREF="https://jenkins.scala-sbt.org:8592/job/sbt-validator/parambuild?sbt=pull/3789/head&zinc=pull/463/head&io=pull/100/head&librarymanagement=pull/189/head&util=pull/134/head&website=1.x&deploy_to_bintray=false&run_sbt_community=false">(restart)</A>

typesafe-tools commented:
A validation involving this pull request <a href="https://jenkins.scala-sbt.org:8592/job/sbt-validator/322/console">is in progress...</a>

typesafe-tools commented:
[//]: # (Refs: pull/3789/head-pull/463/head-pull/100/head-pull/189/head-pull/134/head-1.x)
The validator <A HREF="https://jenkins.scala-sbt.org:8592/job/sbt-validator/322/">has checked</A> the following projects against Scala 2.12,
tested using dbuild, projects built on top of each other.

Project             | Reference            | Commit
:------------------ | :------------------- | :----------------
  sbt               | <A HREF="https://github.com/sbt/sbt/pull/3789">pull/3789/head      </A> | sbt/sbt@30b53618c99d80dae9886651436df233d8b4cb2f
  zinc              | <A HREF="https://github.com/sbt/zinc/pull/463">pull/463/head       </A> | sbt/zinc@422340c8e0abb31a0cc42c9e3ff3c726498c8028
  io                | <A HREF="https://github.com/sbt/io/pull/100">pull/100/head       </A> | sbt/io@8c5947be6c04c6a0b5d6543ec181f10f4b5d6cc7
  librarymanagement | <A HREF="https://github.com/sbt/librarymanagement/pull/189">pull/189/head       </A> | sbt/librarymanagement@aacab10c79abbbe206ab7b6e251a6ab8e6313fdd
  util              | <A HREF="https://github.com/sbt/util/pull/134">pull/134/head       </A> | sbt/util@cd4346c5d7aa0c04893bc7ae85c240e1a6926c62
  website           | <A HREF="https://github.com/sbt/website/tree/1.x">1.x                 </A> | 

:x: The result is: ***FAILED***
<A HREF="https://jenkins.scala-sbt.org:8592/job/sbt-validator/parambuild?sbt=pull/3789/head&zinc=pull/463/head&io=pull/100/head&librarymanagement=pull/189/head&util=pull/134/head&website=1.x&deploy_to_bintray=false&run_sbt_community=false">(restart)</A>

typesafe-tools commented:
A validation involving this pull request <a href="https://jenkins.scala-sbt.org:8592/job/sbt-validator/323/console">is in progress...</a>

typesafe-tools commented:
[//]: # (Refs: pull/3789/head-pull/463/head-pull/100/head-pull/189/head-pull/134/head-1.x)
The validator <A HREF="https://jenkins.scala-sbt.org:8592/job/sbt-validator/323/">has checked</A> the following projects against Scala 2.12,
tested using dbuild, projects built on top of each other.

Project             | Reference            | Commit
:------------------ | :------------------- | :----------------
  sbt               | <A HREF="https://github.com/sbt/sbt/pull/3789">pull/3789/head      </A> | sbt/sbt@5ab232b0ad87a5479078cb4a18fe5599b8b15d36
  zinc              | <A HREF="https://github.com/sbt/zinc/pull/463">pull/463/head       </A> | sbt/zinc@422340c8e0abb31a0cc42c9e3ff3c726498c8028
  io                | <A HREF="https://github.com/sbt/io/pull/100">pull/100/head       </A> | sbt/io@8c5947be6c04c6a0b5d6543ec181f10f4b5d6cc7
  librarymanagement | <A HREF="https://github.com/sbt/librarymanagement/pull/189">pull/189/head       </A> | sbt/librarymanagement@aacab10c79abbbe206ab7b6e251a6ab8e6313fdd
  util              | <A HREF="https://github.com/sbt/util/pull/134">pull/134/head       </A> | sbt/util@cd4346c5d7aa0c04893bc7ae85c240e1a6926c62
  website           | <A HREF="https://github.com/sbt/website/tree/1.x">1.x                 </A> | 

:white_check_mark: The result is: ***SUCCESS***

The binaries are at https://dl.bintray.com/sbt/maven-snapshots
The new sbt version is: 1.1.0-bin-20171214T111843
<A HREF="https://jenkins.scala-sbt.org:8592/job/sbt-validator/parambuild?sbt=pull/3789/head&zinc=pull/463/head&io=pull/100/head&librarymanagement=pull/189/head&util=pull/134/head&website=1.x&deploy_to_bintray=false&run_sbt_community=false">(restart)</A>

typesafe-tools commented:
A validation involving this pull request <a href="https://jenkins.scala-sbt.org:8592/job/sbt-validator/325/console">is in progress...</a>

typesafe-tools commented:
A validation involving this pull request <a href="https://jenkins.scala-sbt.org:8592/job/sbt-validator/326/console">is in progress...</a>

typesafe-tools commented:
[//]: # (Refs: pull/3789/head-pull/463/head-pull/100/head-pull/189/head-pull/134/head-1.x)
The validator <A HREF="https://jenkins.scala-sbt.org:8592/job/sbt-validator/326/">has checked</A> the following projects against Scala 2.12, 2.11,
tested using dbuild, projects built on top of each other.

Project             | Reference            | Commit
:------------------ | :------------------- | :----------------
  sbt               | <A HREF="https://github.com/sbt/sbt/pull/3789">pull/3789/head      </A> | sbt/sbt@5ab232b0ad87a5479078cb4a18fe5599b8b15d36
  zinc              | <A HREF="https://github.com/sbt/zinc/pull/463">pull/463/head       </A> | sbt/zinc@422340c8e0abb31a0cc42c9e3ff3c726498c8028
  io                | <A HREF="https://github.com/sbt/io/pull/100">pull/100/head       </A> | sbt/io@8c5947be6c04c6a0b5d6543ec181f10f4b5d6cc7
  librarymanagement | <A HREF="https://github.com/sbt/librarymanagement/pull/189">pull/189/head       </A> | sbt/librarymanagement@aacab10c79abbbe206ab7b6e251a6ab8e6313fdd
  util              | <A HREF="https://github.com/sbt/util/pull/134">pull/134/head       </A> | sbt/util@cd4346c5d7aa0c04893bc7ae85c240e1a6926c62
  website           | <A HREF="https://github.com/sbt/website/tree/1.x">1.x                 </A> | 

:x: The result is: ***FAILED***
<A HREF="https://jenkins.scala-sbt.org:8592/job/sbt-validator/parambuild?sbt=pull/3789/head&zinc=pull/463/head&io=pull/100/head&librarymanagement=pull/189/head&util=pull/134/head&website=1.x&deploy_to_bintray=false&run_sbt_community=false">(restart)</A>

Commit:
Merge pull request #1 from dwijnand/wip-milli

Use IO.getModified over importing the method

Commit:
Removed a couple more direct imports of getModifiedTime()

eed3si9n committed:
IO 1.1.2

typesafe-tools commented:
A validation involving this pull request <a href="https://jenkins.scala-sbt.org:8592/job/sbt-validator/330/console">is in progress...</a>

typesafe-tools commented:
[//]: # (Refs: pull/3789/head-pull/463/head-pull/100/head-pull/189/head-pull/134/head-1.x)
The validator <A HREF="https://jenkins.scala-sbt.org:8592/job/sbt-validator/330/">has checked</A> the following projects against Scala 2.12, 2.11,
tested using dbuild, projects built on top of each other.

Project             | Reference            | Commit
:------------------ | :------------------- | :----------------
  sbt               | <A HREF="https://github.com/sbt/sbt/pull/3789">pull/3789/head      </A> | sbt/sbt@508947178325304deccab72654ec47eb16dc41c2
  zinc              | <A HREF="https://github.com/sbt/zinc/pull/463">pull/463/head       </A> | sbt/zinc@717b2489d1dac291161234b25068d8a5f75730ee
  io                | <A HREF="https://github.com/sbt/io/pull/100">pull/100/head       </A> | sbt/io@8c5947be6c04c6a0b5d6543ec181f10f4b5d6cc7
  librarymanagement | <A HREF="https://github.com/sbt/librarymanagement/pull/189">pull/189/head       </A> | sbt/librarymanagement@0b496f4d41aadd8910998e69d8ef0e4235fe7155
  util              | <A HREF="https://github.com/sbt/util/pull/134">pull/134/head       </A> | sbt/util@3d9eab1bf8866010b8643261e7f2df3bab1102ab
  website           | <A HREF="https://github.com/sbt/website/tree/1.x">1.x                 </A> | 

:x: The result is: ***FAILED***
<A HREF="https://jenkins.scala-sbt.org:8592/job/sbt-validator/parambuild?sbt=pull/3789/head&zinc=pull/463/head&io=pull/100/head&librarymanagement=pull/189/head&util=pull/134/head&website=1.x&deploy_to_bintray=false&run_sbt_community=false">(restart)</A>
